plugins/file-buffer/README.md

```diff
@@ -0,0 +1,38 @@
+# Flatfile File Buffer Plugin
+
+This plugin will provide a file buffer for any configured file. It will read the file and return it as a nodejs Buffer object. This is a helpful underlying utility for writing file extractor plugins.
```

Comment:
```suggestion
This plugin will provide a file buffer for any configured file. It will read the file and return it as a [nodejs Buffer object[(https://nodejs.org/api/buffer.html). This is a helpful underlying utility for writing file extractor plugins.
```